site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,106 @@
+
+  [#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+       If we have an application context, append the client_id to ensure the correct application theme when applicable.
+  --]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+
+${url}
+
+- FusionAuth Admin
+```
+
+Be sure to save your icon by clicking the blue save icon.
+
+When a user clicks the `Forgot Password` link on the hosted login page or calls the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[forgot password API], this template will be used to email the user to start the workflow to change their password for all applications in the specified Tenant - so long as they don't have a template set at the Application level.
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview]. After, log in to your application by clicking on `Applications` in the navigation sidebar, selecting the green magnifying glass icon, and copying and pasting the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+=== Overriding Email Templates using API
+
+Finally, we'll override our generic Tenant Email Template we just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the blue edit icon for the `Forgot Password` email template and copy the `HTML Template` `Text Template` sections into a notepad.
+
+Create a new email template by going to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste your default template from above into the `HTML Template`  and `Text Template` sections. Feel free to edit these templates based on your application.
+
+Save your template by clicking the blue save icon in the top right corner.
+
+=== Test the Custom Template
+
+For our demo application Pied Piper, we'll call the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Start Forgot Password Workflow] for an existing user. If you haven't already created a user, go ahead and complete the process in the  link:/docs/v1/tech/tutorials/register-user-login-api[Register a User and Login] guide.
+```shell
+curl --request POST \
+  --url https://localhost/api/user/forgot-password \
+  --header 'Authorization: xxxxxx' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "applicationId": "85a03867-dccf-4882-adde-1a79aeec50df",
```

Comment:
Does the user need to be registered for the application? Will the email send (with the application specific template) if they are not?

Reply:
Added a note that the user needs to be registered for the email to be sent. 👍